namespaces.tf

```diff
@@ -0,0 +1,8 @@
+resource "kubernetes_namespace" "ack" {
+  depends_on = [var.mod_dependency]
+  count      = (var.enabled && var.create_namespace && var.namespace != "kube-system") ? 1 : 0
+
+  metadata {
+    name = var.namespace
+  }
+}
```

Comment:
it looks like something is wrong with this code

Reply:
what is that like, whats wrong?

Reply:
code looks great job!